coredns/README.md

```diff
@@ -85,6 +86,7 @@ spec:
 
 **Notes**:
 
+- The shipped `coredns.d/auto_conf.yaml` file enables the legacy version of the check by default.  
```

Comment:
The example on line 76 still uses `openmetrics_endpoint`. I think it can be helpful to clarify that `openmetrics_endpoint` = V2 and `prometheus_url` = legacy/V1 in these notes (and the other README notes).

Reply:
I left these as-is because they're examples for manually configuring autodiscovery.

Reply:
I can add an example for using legacy/V1